<launch>
  <!-- declare args to be passed in -->
  <arg name="obj_descriptor" default='{"type" : "Foo"}'/> 
  <arg name="search_method" default='random'/> 

  <param name="obj_desc" type="string" value="$(arg obj_descriptor)" />
  <param name="search_method" type="string" value="$(arg search_method)" />    
  <param name="min_objs" type="int" value="1" />
  <param name="max_objs" type="int" value="1" />
  <param name="max_time" type="int" value="120" />
  <param name="max_poses" type="int" value="10" />  
  
  <node name="search_agent" pkg="object_search" type="search_agent.py" output="screen"/>
</launch>
